fake_test.go

```diff
@@ -560,7 +560,14 @@ func TestFakeAddInsertReplace(t *testing.T) {
 }
 
 func TestFakeParseDump(t *testing.T) {
-	tc1 := `
+	type tc struct {
+		dump     string
+		ipFamily Family
+	}
+
+	tc1 := tc{
```

Comment:
can you rewrite this to be more like the "standard" unit test style with

```
	for _, tc := range []struct {
		name string
		...
	}{
	...
```

eg like most of the ones in `nftables_test.go`. (The existing tests in `fake_test.go` weren't written that way because they aren't doing a series of independent tests, they're doing a single test with many parts.)